ui/src/components/ChoiceModal.tsx

```diff
@@ -243,25 +243,48 @@ export const DayPickerField : React.FC<{
         );
 }
 
+// convert a JavaScript "Date" value into a DAML-backend "Time" value
+const dateToTime = (d : Date) : Time => d.toISOString();
+
 export const DayTimePickerField : React.FC<{
   name: string
   errors?: ChoiceErrorsType,
 }> = ({ name, errors }) => {
   const [ field, meta, { setValue } ] = useField<Time | Nothing>(name);
+  if (field.value == Nothing) {
+    // can't help but render some picked date, so might as well set it
+    setValue(dateToTime(new Date()));
```

Comment:
This is not legal, unfortunately. It will result in an infinite loop, as `setValue` will cause `DayTimePickerField` to be reevaluated (which calls `setValue` again, and so on and so forth).

Reply:
won't it be `!= Nohinng` the second time so all good?